### PR TITLE
Feat milestone two

### PR DESCRIPTION
**Pull request by canriquez**
Hello, please review this request.

This branch includes all milestone two requirements :

- Directory structure including 4 component files.
- App Component implemented as a functional component. It renders  Display and ButtonPanel components. 
- Display Component. Accepts 'result' prop as string. PropTypes and defaultProps specified. 
- Button Component. It accepts 'name' as prop. 
- Button Panel component. Renders calculator panel with all buttons. 

Thanks
Carlos

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Big from 'big.js';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const x = Big(0.002).toString();
```
Comment (bellom) [inline src/components/App.js]: Please let's carefully read and follow the instructions in the pathwright as we both know that all Microverse students are bind to follow the project requirements and instructions.

I'm curious; I don't think we need this line of code in this file;
According to the instructions in the pathwright, we just need to render the components;
image

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    >
+      {' '}
+      {buttonName}
+      {' '}
+    </button>
+  );
+};
+
+Button.defaultProps = {
+  buttonName: 'H',
```
Comment (bellom) [inline src/components/Button.js]: Hello dev, I'm curious, `buttonName`: 'H', I'm trying to understand where the H is coming from.

I think it should be `butonName: ''`


Kindly explain in detail for the next reviewer to get the full idea, thanks.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @canriquez :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/components/App.js (hunk 1 above)
Comment (canriquez) [reply src/components/App.js]: Thanks for the feedback. 
That line was my initial test for the Big library. 
I believe in this case, I am complying with the requirement. I believe my app is rendering the Display and Button Panel components. The requirements do not say anything about rendering only the default value 0. 
Still, I believe you have a fair point with your interpretation. I have corrected this, and now the app renders the default value to 0 only.

File: src/components/Button.js (hunk 2 above)
Comment (canriquez) [reply src/components/Button.js]: Thanks for the review.
This is an arbitrary letter I used to test the default value rendering. I needed something to take some space in the button when I was testing the rendering. The specs do not say anything about the default value in this case either. 
Again, I have now corrected this point also as I believe you have a fair point. For submission leaving an 'H' by default does rise confusion.
Thanks again.
Carlos

**Pull request merged**